tests/AppBundle/Controller/AdherentControllerTest.php

```diff
@@ -110,14 +110,13 @@ public function testEditAdherentProfile()
         $errors = $crawler->filter('.form__errors > li');
 
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(7, $errors->count());
+        $this->assertSame(6, $errors->count());
         $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(0)->text());
         $this->assertSame('Cette valeur ne doit pas être vide.', $errors->eq(1)->text());
-        $this->assertSame('Cette ville et ce code postal ne sont pas liés.', $errors->eq(2)->text());
+        $this->assertSame('Cette valeur n\'est pas un code postal français valide.', $errors->eq(2)->text());
```

Comment:
I changed the behavior of the rule as it was wrong: we wern't checking properly the country. This is the new expected behavior.